packages/e2e-test-utils/src/visitBlockWidgetScreen.js

```diff
@@ -27,17 +27,23 @@ async function visitBlockWidgetScreen() {
   await visitAdminPage('widgets.php');
 
   // Disable welcome guide if it is enabled.
-  const isWelcomeGuideActive = await page.evaluate(() =>
+  const isWelcomeGuideActive = await page.evaluate(() => {
+    if (!wp.data.dispatch('core/edit-widgets').__unstableIsFeatureActive) {
```

Comment:
I think the reason this is not available in 5.9 is because the Twenty Twenty-Two theme doesn't have any widgets support because of Full Site Editing. You'll get this error page:

![Screenshot 2022-01-06 at 15 28 00](https://user-images.githubusercontent.com/841956/148398131-144da1d1-b9cf-4d0b-98d2-0a53649a0e8f.png)

It would be good to test for this accordingly or at least add an inline comment with this explanation.

Reply:
But we don't use 2022 theme in e2e tests. [See](https://github.com/google/web-stories-wp/blob/58f8648bc2df6a53c91db89d8a434cf25965a39b/bin/local-env/install-wordpress.sh#L152).

Reply:
These unstable data entries were added here https://github.com/WordPress/gutenberg/pull/31925.
Removed in https://github.com/WordPress/gutenberg/pull/33774/

Reply:
Nice find!